spade/core/project.py

```diff
@@ -59,6 +21,9 @@ def __init__(self, dbfile):
         self.__add_info("creation_datetime", datetime.datetime.now(), True)
         self.__add_info("update_datetime", datetime.datetime.now())
 
+    def db(self):
```

Comment:
We never define a _db object, I instead have a function db_engine() that returns the sqlalchemy engine.